sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_device.cpp

```diff
@@ -1075,6 +1072,15 @@ void ZeUSMImportExtension::setZeUSMImport(ur_platform_handle_t_ *Platform) {
         zeDriverGetExtensionFunctionAddress,
         (DriverHandle, "zexDriverReleaseImportedPointer",
          reinterpret_cast<void **>(&zexDriverReleaseImportedPointer)));
+    // Hostptr import/release is supported by this platform.
+    Supported = true;
+
+    // Check if env var SYCL_USM_HOSTPTR_IMPORT has been set requesting
+    // host ptr import during buffer creation.
+    const char *USMHostPtrImportStr = std::getenv("SYCL_USM_HOSTPTR_IMPORT");
```

Comment:
do we still need this env var? if now we have interfaces for the user, then it would be up to the user through their calls to  say if they want to use this feature or not. I dont see value keeping this env var.

Reply:
The env var is for SYCL buffers, different from direct use of the import/release APIs.